crates/task-impls/src/consensus.rs

```diff
@@ -488,6 +489,18 @@ where
             self.cur_view = new_view;
             self.current_proposal = None;
 
+            // Poll the future leader for lookahead
+            let lookahead_view = new_view + LOOK_AHEAD;
```

Comment:
I realize that right now LOOK_AHEAD is 5. As part of the benchmarking of libp2p, I think we should tune this value. I'm very curious whether or not 5 is actually optimal. https://github.com/EspressoSystems/HotShot/issues/1706 I added an issue for this.